.github/workflows/_test_pax.yaml

```diff
@@ -518,12 +481,12 @@ jobs:
           ) | tee $GITHUB_STEP_SUMMARY
 
   outcome:
-    needs: publish-test
+    needs: sitrep
```

Comment:
This job would be unnecessary if we localize the error status to individual test jobs. However, there may still be situations where it is needed, i.e. when some checks are performed on the collective results of many/all test jobs.

Reply:
Got it.